extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -780,6 +780,9 @@ export class AmpStoryEmbeddedComponent {
       dev().assertElement(this.focusedStateOverlay_),
       () => {
         this.focusedStateOverlay_.classList.toggle('i-amphtml-hidden', false);
+        tryFocus(
```

Comment:
I think we need to import `tryFocus` on this file

Reply:
Done